src/lib/rest/RestService.cpp

```diff
@@ -149,7 +149,7 @@ std::string payloadParse
 *
 * tenantCheck - 
 */
-static std::string tenantCheck(const std::string& tenant)
+extern std::string tenantCheck(const std::string& tenant)
```

Comment:
Do we really need "extern" here? (typically, when we use "extern" in the .h, we use nothing in the .cpp...)